user/vpn.rst

```diff
@@ -0,0 +1,143 @@
+VPN-Client Automation
+=====================
+
+
+Installing VPN Server and importing the VPN configuration
+---------------------------------------------------------
+We will be installing VPN Server using `this ansible role <https://github.com/Stouts/Stouts.openvpn>`_
+and  then import the VPN configuration into OpenWISP.
```

Comment:
I would specify that one can also install the VPN server in another way if they feel more comfortable, the important thing is that they get the VPN server running and then they import it into OpenWISP